app/assets/src/components/views/SampleViewV2/SampleViewHeader.jsx

```diff
@@ -105,7 +105,7 @@ export default function SampleViewHeader({
             label: sample.name,
             id: sample.id,
             onClick: () => {
-              openUrl(`/samples/${sample.id}/show_v2`, event);
+              openUrl(`/samples/${sample.id}/show`, event);
```

Comment:
This causes an error when selecting another sample in the project from the dropdown, I think the route is just ``/samples/${sample.id}``

Reply:
Done!